mantidproject/mantid
# Replace JSON Strings with Files in DecoderTest

**Description of work.**
Replaces the JSON strings in `DecoderTest.h` with files that can be used instead. This improves the appearance of the test file while also being more maintainable as the JSON can be read and edited more easily. 

**To test:**
1. Check the tests pass. 

Fixes #33580 


*This does not require release notes* because **it is a change to the tests, so is not user facing.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
